### ISSUE TITLE
Connection issues

### ISSUE THREAD

**Issue by crypjunkie** [hyle-team/zano#110]
So i seem to have came across an intermittent issue with zano on staking, If I have a less than constant/stable internet connection, it seems to be making a mess within my staking of zano, so for an eg... I can loose net connect for say a few minutes, and reconnect again within same timeframe, but my issue lies with the Zano wallet showing as connected and synced yet I may have to restart wallet to get another stake reward after 24-26hrs of noticing no rewards. If I've been down for say 24-36 hr's with a wallet weight of over 100k zano, then do a restart I usually have a new stake reward within minutes.

**Comment by clintar** [comment]
Yeah, similarly wallet just stops trying to connect after rapidly trying to connect to daemon a certain number of times. So I can just start and immediately start the daemon and they never try to connect to the daemon again.

**Comment by cryptozoidberg** [comment]
@clintar I've created new issue with your description #130 
This problem that was mentioned by @crypjunkie  is fixed
